catroid/src/main/java/org/catrobat/catroid/ui/ScratchConverterActivity.java

```diff
@@ -235,28 +227,13 @@ public boolean onCreateOptionsMenu(Menu menu) {
 
 	@Override
 	public boolean onPrepareOptionsMenu(Menu menu) {
-		handleShowDetails(searchProjectsListFragment.getShowDetails(),
-				menu.findItem(R.id.menu_scratch_projects_show_details));
-		return super.onPrepareOptionsMenu(menu);
-	}
-
-	@Override
-	public boolean onOptionsItemSelected(MenuItem item) {
-		switch (item.getItemId()) {
-			case R.id.menu_scratch_projects_convert:
-				Log.d(TAG, "Selected menu item 'convert'");
-				searchProjectsListFragment.startConvertActionMode();
-				break;
-			case R.id.menu_scratch_projects_show_details:
-				Log.d(TAG, "Selected menu item 'Show/Hide details'");
-				handleShowDetails(!searchProjectsListFragment.getShowDetails(), item);
-				break;
-		}
-		return super.onOptionsItemSelected(item);
+		handleShowDetails(getSearchFragment().getShowDetails(),
+				menu.findItem(R.id.show_details));
+		return true;
 	}
 
 	private void handleShowDetails(boolean showDetails, MenuItem item) {
-		searchProjectsListFragment.setShowDetails(showDetails);
+		getSearchFragment().setShowDetails(showDetails);
```

Comment:
does this really have to be a method call? it is two lines of code and you're handing through the menu item, which seems strange. don't you think the item's text could be set in the onPrepareOptionsMenu() method? would make more sense to me, as it is exactly what you'd expect from the method and an additional call is kind of unnecessary.